pkgs/development/python-modules/mdformat-myst/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  mdformat,
+  mdit-py-plugins,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "mdformat-myst";
+  version = "0.1.5";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "executablebooks";
+    repo = pname;
```

```suggestion
    repo = "mdformat-myst";
```
https://github.com/NixOS/nixpkgs/issues/277994